postgresql-client/plan.sh

```diff
@@ -0,0 +1,20 @@
+source ../postgresql/plan.sh
+
+pkg_name=postgresql-client
+pkg_origin=jb
```

Comment:
this should be `core` or bldr will get mad

Reply:
whoops! fixed!